.erb-lint.yml

```diff
@@ -731,7 +731,6 @@ linters:
           - is-overlay-fixed
           - is-pseudo-focused
           - is-searchable
-          - is-selected
```

Comment:
I have found out that compiled version of assets, is actually containing "is-selected".